media-video/webcamoid/webcamoid-9.0.0.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="A full featured webcam capture application"
+HOMEPAGE="https://webcamoid.github.io"
+SRC_URI="https://github.com/webcamoid/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+RESTRICT="mirror"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+CMAKE_USE_DIR="${WORKDIR}/${PF}"
+BUILD_DIR="${CMAKE_USE_DIR}/${PN}-build"
+
+IUSE="alsa coreaudio ffmpeg gstreamer jack libuvc oss pulseaudio qtaudio v4lutils videoeffects debug headers v4l"
+
+REQUIRED_USE="v4lutils? ( v4l )"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtquickcontrols2:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	ffmpeg?	( media-video/ffmpeg:= )
+	gstreamer? ( >=media-libs/gstreamer-1.6.0 )
+	jack? ( virtual/jack )
+	libuvc? ( media-libs/libuvc )
+	pulseaudio? ( media-sound/pulseaudio )
+	qtaudio? ( dev-qt/qtmultimedia:5 )
+	v4l? ( media-libs/libv4l )
+"
+DEPEND="${RDEPEND}
+	>=sys-kernel/linux-headers-3.6
+"
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+	dev-util/cmake
```

Comment:
You're inheriting *cmake.eclass* it will take care of this dep for you.

Reply:
Thanks for the tip!